ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -5621,6 +5625,10 @@ keys:
             new variable takes precedence
 
             '
+          deprecation:
```

Comment:
Remove the deprecation notice in the description, since we generate that automatically from the deprecation feature.
Same elsewhere.

Reply:
Removed the deprecation notices in the descriptions for the old variables.